freqtrade/freqtradebot.py

```diff
@@ -619,6 +619,24 @@ def execute_sell(self, trade: Trade, limit: float) -> None:
         trade.open_order_id = order_id
         trade.close_rate_requested = limit
 
+        # Check trade and Sync Trades table as open_order_id: is set.
+        if trade.open_order_id:
+            # Update trade with order values
+            logger.info('Found open order for %s', trade)
+            order = self.exchange.get_order(trade.open_order_id, trade.pair)
+            # Try update amount (binance-fix)
+            try:
+                new_amount = self.get_real_amount(trade, order)
+                if order['amount'] != new_amount:
+                    order['amount'] = new_amount
+                    # Fee was applied, so set to 0
+                    trade.fee_open = 0
```

Comment:
i think during sell you should set fee_close, not fee_open.

Reply:
Good spot, ill have to check source but that may be a bug in original code I reused! 

These is the chance I reused the wrong section, this PR took 5 attempts before my GIT stopping being one.